catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -844,15 +848,11 @@ private SolrQuery logicalOperator(List<SolrQuery> operands, String operator) {
                 throw new UnsupportedOperationException("Null operand found");
             }
             String operandAsString = operand.toString();
-            try {
-                if (operandAsString.contains(URLEncoder.encode(SCORE_DISTANCE, "UTF-8"))) {
-                    SolrQuery temp = operands.get(0);
-                    operands.set(0, operand);
-                    operands.set(i, temp);
-                    break;
-                }
-            } catch (UnsupportedEncodingException e) {
-                LOGGER.info("Unable to encode {}", SCORE_DISTANCE, e);
+            if (operandAsString.contains(partialUrlEncode(SCORE_DISTANCE))) {
```

Comment:
Solr 5 changed `SolrParams.toString()` to not use URLEncoder but instead uses `org.apache.solr.common.util.StrUtils.partialURLEncodeVal(Appendable dest, String val)`.  I think I am going to change the `operandAsString` to be initialize with `operand.toQueryString()` so I can remove `partialUrlEncode` which might get out of sync when updating Solr and go back to `URLEncoder`.